content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+
+### Game Kit
+ - [FileGameSound.h](https://git.haiku-os.org/haiku/commit/?h=hrev53625&id=69f814cded60c5f977a61a2ab747e4456323e6f2) Allow initialisation of the `BFileGameSound` from a `DataIO` object, to allow for opening game sound files from sources other than files.
+
+### Interface Kit
+ - [Deskbar.h](https://git.haiku-os.org/haiku/commit/?h=hrev52499&id=efafab643ce980e3f3c916795ed302599f6b4f66) Add MaxItemWidth() and MaxItemHeight() as part of the work to support variable size icons for Deskbar replicants.
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=5bbf7f1be073d376e37da34681c8e6c7acfcf198) New `B_MOUSE_BUTTON` macro that helps you identify which mouse button was clicked on a mouse with more than three buttons.
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=47102c074278637c3b33935b451d53f7ad176658) Add a the new `B_TRANSPARENT_BACKGROUND`, to identify views that have a transparant background (i.e. the background will not be drawn for those views).
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=c67dde0f2ccb45d550af410cf6d73fee39d10504) Add Tiled Background methods to `BView` objects to asynchronously draw bitmaps as background tiles for the view, using the overloaded `DrawTiledBitmapAsync()` method.
+
+### Locale Kit
+ - [Country.h](https://git.haiku-os.org/haiku/commit/headers/os/locale/Country.h?id=70cdd7d4f5fc62e8b3e220646f84235ec3d444d5) Add a `SetTo()` and an `InitCheck()` method to the `BCountry` class.
+
+### Package Kit
+ - [PackageRoster.h](https://git.haiku-os.org/haiku/commit/headers/os/package/PackageRoster.h?id=e2c7bb900cf83fe0a3834ad2189577687ffaa03f) Add a `IsRebootNeeded()` method to the `BPackageRoster` class.
+ - [PackageRoster.h](https://git.haiku-os.org/haiku/commit/headers/os/package/PackageRoster.h?id=b711002d345e2e9e0075be46a37492624f73ccdd) Add a `IsPackageActive()` method to the `BPackageRoster` class.
+
+## New contributor
+
+Since the last release, Kyle Ambroff-Kao received commit access.
+
+## Source code
+
+The source code of Haiku itself, the source code of the required build tools and the optional packages (except for closed source ones) is contained within the release images as the `_source` packages (except on the "CD" image, which it was left out of due to space constraints.)
+
+## Reporting issues
+
+There are almost  3200 open tickets on Haiku's bug tracker and over 12700 closed items.  If you find what you believe to be an issue, please search our Trac to see if it has already been reported, and if not, file a new ticket at <https://dev.haiku-os.org/>.
+
+For information about major issues that have been fixed since the release, visit <https://dev.haiku-os.org/wiki/R1/Beta2/ReleaseAddendum>.
+
+For more help see the 'Welcome' link on the Haiku desktop, or visit the Haiku Project's website at <https://www.haiku-os.org>.
```

Comment:
There is no link to the welcome page on the desktop anymore. It is however still the start page of webpositive.